### ISSUE TITLE
3DS: Minish Cap hangs on save

### ISSUE THREAD

**Issue by MarioKartFan** [mgba-emu/mgba#1556]
I am playing on a Nintendo 2DS XL.  I love mGBA; however, whenever I save the game in Legend of Zelda: The Minish Cap, it takes nearly a full minute to write the save file (as opposed to just a few seconds if I am using my original GBA cartridge).  This issue does not occur in other emulators using the same ROM file. 

The issue occurs on the latest beta.  It also occurs on every one of the 0.7 versions that were released.  I have no issues saving in other GBA games.

**Comment by endrift** [comment]
I'm not able to reproduce this. Seems to save plenty fast for me.

**Comment by MarioKartFan** [comment]
Thanks. Are you testing a 3DS?

**Comment by MarioKartFan** [comment]
I tested again using the European version. I have the same issue. When using in game saves it hangs for about 35 seconds. It eventually resumes. Save states work fine though.

**Comment by endrift** [comment]
Yes, I was testing on a 3DS. Are you using the CIA version or the 3DSX version? Do you have Luma installed or are you using a non-CFW entrypoint?

**Comment by MarioKartFan** [comment]
I am using the CIA version. I have Luma installed (latest public release).
